ovs: Raise invalid argument error when desired OVS bridge with MAC

cathay4t opened the pull request:
When user desired OVS bridge with MAC address defined, they will get
Verification as OVS bridge does not have MAC address after applied.

This patch raise `InvalidArgument` error in sanitize stage with
suggestion to use OVS internal interface as error message.

Unit test case included.

liangwen12year commented:
Looks good to me, the MAC address on an OVS bridge is influenced by the MAC address of the port or the internal interface that is added to the bridge.

ihuguet commented on rust/src/lib/ifaces/ovs.rs:
```diff
@@ -124,6 +124,19 @@ impl OvsBridgeInterface {
+                return Err(NmstateError::new(
+                    ErrorKind::InvalidArgument,
+                    format!(
+                        "OVS Bridge {} could not hold MAC address \
```
could -> can. Also, add a comma or a dot between "address" and "please".

ihuguet replied:
Apart from this nit, LGTM

cathay4t committed:
ovs: Raise invalid argument error when desired OVS bridge with MAC

When user desired OVS bridge with MAC address defined, they will get
Verification as OVS bridge does not have MAC address after applied.

This patch raise `InvalidArgument` error in sanitize stage with
suggestion to use OVS internal interface as error message.

Unit test case included.

Signed-off-by: Gris Ge <fge@redhat.com>
